src/components/contactList/contactList.jsx

```diff
@@ -2,25 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ContactItem from '../contactItem/contactItem';
 
-const ContactList = ({ contacts, onDeleteClick }) => (
-  <React.Fragment>
-    {(contacts.length === 0) && <p className="mt-4">No results found :(</p>}
-    <table className="table mt-4">
-      <tbody>
-        {contacts.map(item => (
-          <ContactItem
-            id={item.id}
-            key={item.id}
-            name={item.name}
-            phone={item.phone}
-            onDeleteClick={onDeleteClick}
-          />
-        ))
-        }
-      </tbody>
-    </table>
-  </React.Fragment>
-);
+const ContactList = ({ contacts, onDeleteClick }) => {
+  const contactList = contacts.map((contact) => {
+    const obj = {
+      id: contact.id,
+      name: contact.name,
+      phone: contact.phone,
+      onDeleteBtnClick: onDeleteClick,
+    };
+
+    return <ContactItem itemProps={obj} key={contact.id} />;
```

Comment:
Wrong prop name. `contact` would be better

Reply:
Fixed